sw/device/silicon_creator/lib/drivers/otp_unittest.cc

```diff
@@ -64,20 +65,12 @@ TEST_F(OtpReadTest, ReadLenN) {
   }
 
   std::vector<uint32_t> arr(16);
-  EXPECT_EQ(otp_read(0, &arr[0], arr.size() * sizeof(uint32_t)), kErrorOk);
+  otp_read(0, &arr[0], arr.size());
```

Comment:
nit: WDYT about `std::array<uint32_t, 16> arr;` here and `std::array<uint32_t, 16> expected;` below?

```suggestion
  otp_read(0, arr.data(), arr.size());
```

Reply:
I think that's a fine idea!